### DISCUSSION TITLE
How to set minify css not to convert rgba to hex

### DISCUSSION THREAD

**Discussion by panhezeng** [quasarframework/quasar#9794] [Ideas / Proposals]
https://github.com/tdewolff/minify/issues/238

It has indeed appeared that adding support for the Color Module Level 4 was too soon. After reviewing the draft's status (at https://www.w3.org/Style/CSS/current-work), it is clear that it is still in the exploring phase and cannot be expected to have been implemented by all clients. In particular, Edge does not support the #RRGGBBAA notation yet. Existing such notations will be minified using the draft specification though.

**Comment by hawkeye64** [comment]
I am moving this to Discussions...

**Reply by panhezeng** [reply]
```
        chain.optimization
          .minimizer("css").use(require("css-minimizer-webpack-plugin"), [{
          parallel: true,
          minimizerOptions: {
            preset: [
              "default",
              {
                colormin: false
              }
            ]
          }
        }]);
```

**Reply by jiaxincui** [reply]
good!
